eclair-core/src/test/scala/fr/acinq/eclair/channel/states/e/NormalStateSpec.scala

```diff
@@ -2002,19 +2037,23 @@ class NormalStateSpec extends TestKitBaseClass with FixtureAnyFunSuiteLike with
   test("recv Shutdown (with invalid final script)") { f =>
     import f._
     bob ! Shutdown(ByteVector32.Zeroes, hex"00112233445566778899")
-    bob2alice.expectMsgType[Error]
-    bob2blockchain.expectMsgType[PublishTx]
-    bob2blockchain.expectMsgType[PublishTx]
-    bob2blockchain.expectMsgType[WatchTxConfirmed]
-    awaitCond(bob.stateName == CLOSING)
+    bob2alice.expectNoMessage(3 seconds)
```

Comment:
3 seconds feels a lot, 500 millis should be more than enough, isn't it? Same for the tests below.